### ISSUE TITLE
Support utf-8 for IE

### ISSUE THREAD

**Issue by realkyo** [react-csv/react-csv#60]
I need support Chinese, but IE browser can't see right word. 
I use below to work OK. Need a new release, thanks.

    handleLegacy(evt, data, headers, separator, filename) {
    // If this browser is IE 11, it does not support the `download` attribute

      if (window.navigator.msSaveOrOpenBlob) {
        // Stop the click propagation
        evt.preventDefault()

        //let blob = new Blob([toCSV(data, headers, separator)])
        let blob = new Blob(['\uFEFF', toCSV(data, headers, separator)])
        window.navigator.msSaveBlob(blob, filename)

        return false
      }
    }

**Comment by kanashkin-fv** [comment]
May be this way? I put a patch in attach. 
@abdennour, how can i make pull request?

[Fix_issue_#60,_parameter__uFEFF__in__handleLegacy__method_to_add_byte_order_mark.patch.zip](https://github.com/abdennour/react-csv/files/1983722/Fix_issue_.60._parameter__uFEFF__in__handleLegacy__method_to_add_byte_order_mark.patch.zip)


```
handleLegacy(evt, data, headers, separator, filename, uFEFF) {
    // If this browser is IE 11, it does not support the `download` attribute
    if (window.navigator.msSaveOrOpenBlob) {
      // Stop the click propagation
      evt.preventDefault();

      const csv = toCSV(data, headers, separator);
      const blob = new Blob([uFEFF ? '\uFEFF' : '', csv], {type: 'text/csv'});
      window.navigator.msSaveBlob(blob, filename);

      return false
    }
  }

  render(){
    const {data, headers, separator, filename, uFEFF, children , ...rest} = this.props;
    return (
      <a download={filename} {...rest}
         ref={link => (this.link = link)}
         href={this.buildURI(data, uFEFF, headers, separator)}
         onClick={evt => this.handleLegacy(evt, data, headers, separator, filename, uFEFF)}>
        {children}
      </a>
    )
  }

```

**Comment by AddictArts** [comment]
+1

Thanks

@kanashkin-fv Pull request summary: Fork the project. Create a branch, apply your patch, commit push barnch. Then use the pull requests "tab" for this repository and follow the prompts. Click pull requests up top and look at some in the queue.

**Comment by vksantoshsunny** [comment]
Hi

I could not get the the IE download working. I could see that the code is internally handling that using handleLegacy() but I cant get it working on IE 11. Any thoughts?

**Comment by mccabemj** [comment]
@khumphrey PR has been merged #186 addressing this
